### DISCUSSION TITLE
Native functions for DLR and ICF

### DISCUSSION THREAD

**Discussion by DaydreamingElephant** [open2c/cooltools#540] [Ideas]
Greetings! DLR (distal -to-local ratio) and ICF (interchromosomal fraction) are parameters used to characterize chromatin compaction in regions of Hi-C maps. They are supported by other research software. I think that cooltools can benefit from built-in functions that will return binned tracks of these parameters. 
![DLR ICF](https://github.com/user-attachments/assets/86fedb23-c173-4fd6-8647-00d7b68c9f36)

**Comment by Phlya** [comment]
ICF is basically part of `cooltools coverage`, since it calculate both cis and total coverage.
DLR is not directly implemented, but there is this in the sandbox, which will let you calculate it easily: https://github.com/open2c/cooltools/blob/master/cooltools/sandbox/cross_score.py
